setup.py

```diff
@@ -37,4 +37,8 @@
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
     ],
+    data_files=[
+        ('/etc/fedmsg.d/', ['fedmsg.d/mts.py']),
```

Comment:
This should be removed since `/etc/fedmsg.d/` is a volume in the Dockerfile. The configuration should be mounted in the container, not be maintained upstream.

Reply:
This is to install the config file for a common Python package installation rather than running inside container only. Both mounting it to an external volume or without mounting the `/etc/fedmsg.d`, it works.